src/SDKs/SignalR/Management.SignalR/Generated/Models/Resource.cs

```diff
@@ -0,0 +1,72 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This should not have been generated, did we add the `x-ms-azure-resource` flag in the spec